uui-editor/src/plugins/imagePlugin/ImageElement.tsx

```diff
@@ -54,6 +54,8 @@ export function ImageElement({
         let captionEnabled = false;
         if (currentWidth && typeof currentWidth === 'number') {
             captionEnabled = currentWidth >= MIN_CAPTION_WIDTH;
+        } else if (currentWidth && (typeof currentWidth === 'string')) {
+            captionEnabled = true;
```

Comment:
We have a requirement that caption should not appear for images that have width lower than MIN_CAPTION_WIDTH. I guess with your fix, it will not work. I supposed that for this logic it's doesn't matter if which typeof of the width property, we everytime can cast in to number. I guess problem here that we didn't set initial image width on loading, could you check it?

Reply:
Ok let me update this. I will delete this PR and Raise a New One. Thanks

Reply:
@AlekseyManetov

Reply:
@shubhamt619 you can continue in this one, it's not a problem

Reply:
@AlekseyManetov  Thank you for being patient, I have updated the code. Could you please Accept the PR if it's upto the standards and our requirements ?

Reply:
Seems like caption is not shown after insert stil

Reply:
@shubhamt619 i don't see any code updates from your since last review

Reply:
I was going through the react slate documentation to find the solution. I will work on this today and revert as soon as I get the response.

Reply:
@dimaanj @AlekseyManetov Please check now